src/main/java/com/applicaster/playerevents/PlayerEventsProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.applicaster.playerevents;
+
+import java.util.HashMap;
+
+public interface PlayerEventsProvider {
+
+    void onPlayerEvents(String event, HashMap<String,Object> properties);
```

Comment:
Can you name it onPlayerEvents in singular, without the s, as we are receiving just one event.

Reply:
will be fixed